setup.py

```diff
@@ -31,9 +35,21 @@ def add_alias_unix():
     elif user_os == "Linux" and os.path.isfile(BASH_PROFILE_PATH):
         write_to_file(BASH_PROFILE_PATH)
 
+
 def add_alias_windows():
     subprocess.check_call("setx ESLINT_NO_DEV_ERRORS true",
-                             stderr=subprocess.STDOUT, shell = True)
+                          stderr=subprocess.STDOUT, shell=True)
+
+
+# This function is neccesary such that users from different operating systems can
+# both make use of hot reload
+# CHOKIDAR_USEPOLLING and WATCHPACK_POLLING are alternate methods of watching for changes on files
+# windows users need this because the docker containers are all running linux and since their
+# files are also being managed by windows it gets messy using the os default file watching protocal
+# the .env file will be user specific and not interfere with non-windows users
+def make_env(isWindows):
+    f = open(".env", "x")
+    f.write(f"CHOKIDAR_USEPOLLING={isWindows}\nWATCHPACK_POLLING={isWindows}")
```

Comment:
you need to close this file, or you could do
```suggestion
def make_env(isWindows):
    with open(".env", "x"):
        f.write(f"CHOKIDAR_USEPOLLING={isWindows}\nWATCHPACK_POLLING={isWindows}")
```